tests/log/Log.cpp

```diff
@@ -165,7 +165,6 @@ TEST_CASE("Testing the Log<>() function", "[api][log]") {
     expected_count += levels.size() * (levels.size() + 1) / 2;  // Direct reaction logs
     expected_count += levels.size() * (levels.size() + 1) / 2;  // Indirect reaction logs
     expected_count += levels.size() * levels.size();            // Non reaction logs
-    expected_count += 2 + levels.size();                        // Post shutdown logs
```

Comment:
What happened to post shutdown logs?

Reply:
logs only work if `powerplant != nullptr`, but powerplant destruction sets the singleton to `nullptr`, not sure why these ever worked tbh

Reply:
Powerplant destruction sets them to nullptr, but these were on shutdown, not destruction. Powerplant destruction should happen when there is nothing left, no reactors everything has unbound etc. There should be nothing left to log when it goes nullptr

Reply:
No, these were post shutdown, as the comment says

Reply:
hmm hang on, i think I missed something

Reply:
We stopped processing all submitted tasks (even direct/immediate tasks) after `powerplant.shutdown()` was called, so anything emitted after shutdown was ignored. This should be fixed now